### ISSUE TITLE
Option to append more files

### ISSUE THREAD

**Issue by schrmh** [mwh/dragon#65]
When dragon-drop is called multiple times, it opens up a new window every time.
It would be nice if it there was an option to just append the file list.

**Comment by WorldsEndless** [comment]
I agree. It would be handy to put multiple files in the Dragon list, either all at once or by appending asyncronously.

**Comment by timblaktu** [comment]
I'm using dragon 1.2.0 which has 2 options that the manpage implies do what you're asking here (and what I also want - to select and drag multiple files at once):

>        -a --all
>               drag all files at once.
> 
>        -A --all-compact
>               drag all files at once, only displaying the number of files.

But when I `dragon-drop -a [FILES]` each file still appears as an individually-draggable entity in the window, and dragging only drags one file at a time. 

This is confusing, and seems to be a bug, but I thought I'd check with you two about this first - @schrmh @WorldsEndless ...

**Comment by timblaktu** [comment]
I think my problem may be that I'm on Wayland. I discovered that I'm getting these errors in the terminal when running dragon:

```
[130]tim@thinky:~/src > dragon-drop -a kubernetes/community/icons/svg/control_plane_components/labeled/*.svg
Gtk-Message: 12:25:55.828: Failed to load module "window-decorations-gtk-module"
Gtk-Message: 12:25:55.829: Failed to load module "colorreload-gtk-module"

(dragon-drop:181685): Gtk-WARNING **: 12:25:55.840: Could not load a pixbuf from /org/gtk/libgtk/icons/16x16/actions/text-x-generic.png.
This may indicate that pixbuf loaders or the mime database could not be found.
```

I'm on arch + hyprland and it seems that dragon-drop may be hard-coded to only work with GTK?

**Comment by timblaktu** [comment]
Found [this thread](https://www.reddit.com/r/wayland/comments/1184f7k/fixed_this_issue_when_switching_to_wayland/) which fixed my issue with `dragon-drop -a` not working right. 

I guess that `--all` behavior is different from what this issue is asking for.
